igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/Front50Config.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2014 Netflix, Inc.
```

Comment:
This Front50Config and the Front50Service files are replicated from some existing files with "Copyright 2014 Netflix, Inc." so l leave it instead of replacing it to the Oracle one. And I will append another line of Oracle copyright.